src/components/table/OcTable.vue

```diff
@@ -31,7 +31,7 @@
       >
         <oc-td
           v-for="field in fields"
-          :key="`oc-tbody-tr-${item[idKey] || index}-${field.name}`"
+          :key="cellKey(field, index, item)"
```

Comment:
```suggestion
          :key="'oc-tbody-tr-' + cellKey(field, index, item)"
```
I would like to keep the key prefixes consistent with all the other keys in this component.